sunnyportal/responses.py

```diff
@@ -395,3 +396,27 @@ def parse_entry(self, entry, timestamp, converter):
             return None
 
         return EnergyBalance(timestamp, consumption, generation)
+
+
+class LogbookResponse(ResponseBase):
+    def parse(self, data):
+        self.entries = []
+        for e in super().parse(data).iterfind("entry"):
+            device = self.find_or_raise(e, "device")
+            description = self.find_or_raise(e, "description").text
+            description = unescape(description, {"&apos;": "'", "&quot;": '"'})
+            self.entries.append(
+                {
+                    "event_id": self.get_or_raise(e, "event-id"),
+
+                    "date": self.find_or_raise(e, "date").text,
```

Comment:
This should be parsed into a datetime using `datetime.strptime`.

Reply:
Will do.